src/main/java/frc/robot/subsystems/NoteVision.java

```diff
@@ -194,32 +194,38 @@ private void initializeSimulation() {
                 FieldConstants.NOTE_C_3,
                 FieldConstants.NOTE_C_4,
                 FieldConstants.NOTE_C_5,
+                // Blue Stage Notes
                 FieldConstants.NOTE_S_1,
                 FieldConstants.NOTE_S_2,
-                FieldConstants.NOTE_S_3)) {
+                FieldConstants.NOTE_S_3,
+                // Red Stage Notes
+                FieldConstants.flipTranslation(FieldConstants.NOTE_S_1),
+                FieldConstants.flipTranslation(FieldConstants.NOTE_S_2),
+                FieldConstants.flipTranslation(FieldConstants.NOTE_S_3)
+                )) {
```

Comment:
We don't need to have the Bllue Stage notes and the Red Stage notes in this list. You only need one or the other. FieldConstants.flipTranslation() checks to see if we're Red or Blue. If we're Blue when this is called, the last three elements inthe list are a repeat. If we're Red, we don't care about the Blue notes anyway. I would recommend deleting lines 198-201.

Reply:
If we don't know whether we're red or blue, then the code is not going to work since we won't know what FieldConstants.isRedAlliance() will return.

Reply:
No that is a separate issue. Red vs Blue *is* determined once we start up driving or an auto. It is *not* determined during all the construction code, including setting up the simulation.

Reply:
Should be fixed now. I created RED stage notes in FieldConstants.